src/Core/Sts/Input/AssumeRoleRequest.php

```diff
@@ -198,58 +198,53 @@ public function requestBody(): string
         $payload['RoleSessionName'] = $this->RoleSessionName;
 
         (static function (array $input) use (&$payload, $indices) {
-            $indices->kd8fbed2 = 0;
+            $indices->kfc822c1 = 0;
             foreach ($input as $value) {
-                ++$indices->kd8fbed2;
+                ++$indices->kfc822c1;
 
                 if (null !== $value) {
                     (static function (PolicyDescriptorType $input) use (&$payload, $indices) {
                         if (null !== $v = $input->getarn()) {
-                            $payload["PolicyArns.{$indices->kd8fbed2}.arn"] = $v;
+                            $payload["PolicyArns.member.{$indices->kfc822c1}.arn"] = $v;
```

Comment:
this is the bug spotted in Sts tests

Reply:
Awesome. That is the one you were mentioning before but I didnt understand =)